samtools.wdl

```diff
@@ -136,34 +176,50 @@ task Merge {
     }
 }
 
-task SortByName {
+task Sort {
     input {
-        File bamFile
-        String outputBamPath = "namesorted.bam"
+        File inputBam
+        String outputPrefix
+        Boolean sortByName = false
+        String outputFormat = "BAM"
```

Comment:
I think this is better hardcoded into bam.
Otherwise the output  extension (.bam) does not makes sense.
No user input :wink:

Reply:
Additionally: if you use BAM as output, you can also set the output compression level. This should be set to 1. It might use 6 by default which is extremely slow.